pkg/parser/parser.go

```diff
@@ -567,6 +570,17 @@ func (p *Parser) parseBlockWithEndTokens(scope *scope, endTokens map[lexer.Token
 }
 
 func (p *Parser) advance() {
```

Comment:
This function should now probably be renamed, but to what?
`advanceSkippingWS()` that's a mouth full...

Reply:
yeah, I thought about this, but advancePastNL means: keep on advancing until you hit NL and then one more. In this case we want to _only_ advance if there is WS....